pkg/channel/consolidated/reconciler/controller/kafkachannel.go

```diff
@@ -620,3 +619,17 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, kc *v1beta1.KafkaChannel)
 	r.consumerGroupWatcher.Forget(string(kc.ObjectMeta.UID))
 	return newReconciledNormal(kc.Namespace, kc.Name) //ok to remove finalizer
 }
+
+func makeAdminClient(ctx context.Context, clientID string, kafkaAuthCfg *client.KafkaAuthConfig, kafkaConfig *utils.KafkaConfig) (sarama.ClusterAdmin, error) {
+	saramaConf, err := client.NewConfigBuilder().
+		WithDefaults().
+		WithAuth(kafkaAuthCfg).
+		WithClientId(clientID).
+		FromYaml(kafkaConfig.SaramaSettingsYamlString).
```

Comment:
Can we just update the `MakeAdminClient` to have that option passed in? or not, if `nil`? 

Feels a bit weird to duplicate that code

Reply:
Not duped anymore. Moved into a common place